src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -371,11 +379,21 @@ def perform_post_processing(
                 obs_data[col] = obs_data[col].astype(
                     pd.CategoricalDtype(categories=state_categories)
                 )
+            processed_results[observer] = obs_data
+    # Format 1040 by combining with tax dependents to match guardians with dependents
+    processed_results[metadata.DatasetNames.TAXES_1040] = formatter.format_1040_dataset(
+        processed_results
+    )
 
-        logger.info(f"Writing final {observer} results.")
-        obs_dir = build_output_dir(final_output_dir, subdir=observer)
-        seed_ext = f"_{seed}" if seed != "" else ""
-        write_to_disk(obs_data.copy(), obs_dir / f"{observer}{seed_ext}.{extension}")
+    # Write results for each dataset - we do not need to write out tax dependents now that we format
+    # the 1040 dataset above
+    for observer in FINAL_OBSERVERS:
+        if observer in OUTPUT_DATASETS:
+            obs_data = processed_results[observer]
```

Comment:
You also already have obs_data loaded (though I think you did this b/c the 1040 stuff has changed since initial loading?)

Reply:
Yes